slack-styles.user.css

```diff
@@ -34,3 +34,18 @@
     height: 32px;
     width: 32px;
 }
+
+/* Clearer "New Messages" divider */
+.c-message_list__unread_divider__label {
+    color: #ecb22f;
+}
+.c-message_list__unread_divider__separator {
+    border-color: #ecb22f;
+}
+
+/* Font updates */
```

Comment:
Font is entirely a user choice I guess 🤷‍♂️ 

It updates the sidebar since the `#client-ui` element is the top level wrapper for all the content of the Slack app.
Happy to change it to just message content if that's preferred.